sampleWorkspace/.gitignore

```diff
@@ -0,0 +1 @@
+tests/
```

Comment:
Instead of adding a `.gitignore` to the repository you could also ignore the folder locally in your workspace.
I usually do this by adding a `.gitignore` containing only a single * into the folder I want to ignore (`tests` in this case).